lms/services/vitalsource/client.py

```diff
@@ -77,32 +80,39 @@ def parse_document_url(document_url):
     def generate_document_url(book_id, cfi):
         return f"vitalsource://book/bookID/{book_id}/cfi/{cfi}"
 
-    def get_launch_params(self, document_url, lti_user: LTIUser) -> Tuple[str, Optional[Dict[str, str]]]:
+    def get_iframe_url(self, document_url: str) -> str:
+        """
+        Return an URL like https://hypothesis.vitalsource.com/books/{book_id}/cfi/{cfi}.
+
+        That URL can be used to load VitalSource content in an iframe like we do with other types of content.
+
+        Note that this method is an alternative to `get_launch_params` below.
+
+        :param document_url: `vitalsource://` type URL identifying the document
+        """
+        url_params = self.parse_document_url(document_url)
+        return f"https://hypothesis.vitalsource.com/books/{url_params['book_id']}/cfi/{url_params['cfi']}"
+
+    def get_launch_params(
+        self, document_url, lti_user: LTIUser
+    ) -> Tuple[str, Dict[str, str]]:
```

Comment:
Might be worth noting that the LTI launch method is deprecated and we plan to drop it soon.